terraform/tfschema/customtype.go

```diff
@@ -93,8 +93,8 @@ func SchemaStrings() *schema.Schema {
 	}
 }
 
-// GetBoolOption unmarshals bool value
-func GetBoolOption(path string, target reflect.Value, meta *accessors.SchemaMeta, sch *schema.Schema, data *schema.ResourceData) error {
+// FromTerraformBoolOption reads BoolOption from Terraform state
```

Comment:
OK, I get that. But when called from _outside_ this package, it will always have a `tfschema` prefix, which is doubled up in the function name (e.g. `tfschema.FromTerraformBoolOption()`) - the _terraform_ part is already there. If you need to specify _anything_, it should probably the non-terraforn src/dst type (e.g. imagine a `tfschema.LabelFromTeleportLabel()`)

Anyway, I'm nitpicking and it's not something I'm going to bounce this over.